infra/eligibility_screener/template/ecs.tf

```diff
@@ -1,26 +1,22 @@
-# load balancer to redirect
-# resource "aws_lb" "screener" {
-#   name = "${var.environment_name}-screener-lb"
-#   internal = false
-#   load_balancer_type = "application"
-#   security_groups = [aws_security_group.allow-screener-traffic.id]
-# }
-# resource "aws_lb_listener" "screener" {
-
-# }
 # security group for screener
 resource "aws_security_group" "allow-screener-traffic" {
   name        = "allow_screener_traffic"
   description = "This rule blocks all traffic unless it is HTTPS for the eligibility screener"
   vpc_id      = module.constants.vpc_id
 
   ingress {
+    description = "HTTP traffic from VPC"
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/8"]
+  }
```

Comment:
I think you can probably continue to serve traffic on 3000 and consider the routing for 80/443 (HTTP/HTTPS) in the future to route to port 3000. In a traditional environment, you'd have a load balancer or gateway that receives requests, handles certificate verification, and redirects 80/443 to port 3000 on your ECS service.